lib/addons/p5.dom.js

```diff
@@ -47,8 +47,9 @@
    *
    * @method select
    * @param  {String} name id, class, or tag name of element to search for
-   * @param  {String} [container] id, p5.Element, or HTML element to search within
-   * @return {Object|p5.Element|Null} p5.Element containing node found
+   * @param  {String|p5.Element|HTMLElement|null} [container] id, p5.Element, or
```

Comment:
yes indeed. these changes are entirely due to FES errors during test runs.

Reply:
Yeah, I see how it could inadvertently happen.. but I think this documentation could be confusing for people because they would not call the function like this:
`var c = select('#foo', null);`

Reply:
Ok, I'll remove that example, and make passing null an error.

Reply:
ok, i think I came up with something better. this way the `null` case is still an error, and the example is mostly intact.